setupTests.js

```diff
@@ -46,6 +46,9 @@ jest.mock('axios', () => {
   };
 });
 
+// TODO: Remove after all migrations
+jest.mock('src/script/page/RightSidebar/utils/toggleRightPanel', () => () => jest.fn());
```

Comment:
Please move this mock in the file that need it, if we mock it globally we will endup having weird behavior when we want to actually test it